kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -23,6 +23,9 @@ enum AllocData<'a> {
 }
 
 impl<'tcx> GotocCtx<'tcx> {
+    /// A MIR operand is either a constant (literal or `const` declaration) or a place
```

Comment:
Here is an example that I think you should first document what `codegen_operand` does. After that you can add more details on what the operand argument actually is.

Reply:
I tend to leave that out when the name (and maybe type) is really good. What would you like to say about this function besides, uh, "codegens an operand" :)

Reply:
Something like "generate a goto expression that represents a MIR operand."